content/en/integrations/guide/azure-manual-setup.md

```diff
@@ -135,6 +135,49 @@ azure role assignment create --objectId <OBJECT_ID> -o "Monitoring Reader" -c /s
 
 ### Integrating through the Azure portal
 
+{{< tabs >}}
+{{% tab "ARM template" %}}
+
+1. In the Azure integration tile, select **Configuration** > **New App Registration** > **Using Azure Portal**.
+
+2. Select Management Group (Auto-Discover) or Individual Subscriptions. If you select Management Group, Datadog automatically discovers and monitors any subscriptions within that selected scope, including any subscriptions created in the future. You must have the owner role in the Management Group selected.
+
```

Comment:
Also, If they select `Individual Subscriptions` then they need have the `owner` role in the subscriptions they selected.